src/components/Tabber/Tabber.tsx

```diff
@@ -15,27 +17,49 @@ import Tabs from './Tabs/Tabs';
 
 import { Tab, TabId, Tabs as TabsType } from './Tabber.types';
 
+const makeId = (account: AccountPost): `${string}-${string}` =>
+  `${account.id}-${account.socialMediaId}`;
+
 function Tabber(): ReactNode {
-  const { accounts, socialMedias } = useSocialMediaStore();
+  const { accounts } = useAccountStore();
+  const { socialMedias } = useSocialMediaStore();
 
   const [currentTab, setCurrentTab] = useState<TabId>(
-    `${accounts.data?.[0]?.id}-${accounts.data?.[0]?.socialMediaId}`
-  );
-
-  const [tabs, setTabs] = useState<TabsType>(
-    accountsToTabs(accounts.data ?? [], socialMedias)
+    makeId(accounts[0] || {})
   );
-
-  const currentPostMode = socialMedias
-    .get(currentTab.split('-')[1])
-    ?.postModes.find(
-      (postMode: PostMode) => postMode.id === tabs[currentTab].postModeOnView
+  const [tabs, setTabs] = useState<TabsType>({});
+
+  useEffect(() => {
+    if (accounts.length > 0) {
+      const initialTabId = accounts.length > 0 ? makeId(accounts[0]) : '';
+      setCurrentTab(initialTabId as TabId);
+      setTabs(accountsToTabs(accounts, socialMedias));
+    } else {
+      setTabs({});
+      setCurrentTab('' as unknown as TabId);
```

Comment:
Necessário esse unknown? Por que?
```suggestion
      setCurrentTab('' as TabId);
```